### PR TITLE
Adding Sonar Badge and adding sonar host url

### PR DESCRIPTION
**Pull request by biswassri**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,6 +82,9 @@
 
         <!-- CVE-2021-44228 Log4Shell, CVE-2021-4104, CVE-2021-45105, CVE-2021-44832 -->
         <log4j.version>2.17.1</log4j.version>
+
+        <!-- Sonar Setting -->
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
```
Comment (MikeEdgar) [inline pom.xml:87]: I don't think it's required, but for good measure, can you add the sonar.organization also?
```suggestion
        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
        <sonar.organization>bf2fc6cc711aee1a0c2a</sonar.organization>
```

Comment (biswassri) [reply pom.xml:87]: Sure!

**Comment by MikeEdgar** [comment]
> Once merged and analysis shows up, I can add it in

Sounds good, thank you :+1:

**Review by MikeEdgar — APPROVED** [review]

**Review by rareddy — APPROVED** [review]
lgtm

**Pull request merged**